### PR TITLE
Migrate asset_hyperlink docs

### PR DESCRIPTION
**Pull request by davidrapson**
Migrate asset_hyperlink docs to the new format. Applies haml-lint rubocop checks to the file and moves the styles out of the generic "misc" file.

### CODE REVIEW

**Comment by davidrapson** [comment]
@luke-hill Tagged this test as `@future_release` as it's moving to the new format. Any way we can update this ahead of time?

**Review by davidsauntson — COMMENTED** [review]
👍 

Small nitpick re: accessibility.  Voice over reads `mb` as millibars, but will read `MB` as megabytes.  This might just be macOS behaviour, but as `MB` is the standard unit for megabytes we should probably use it.  I know you haven't changed it but seeing as you're there... 😄 

Q: does the type actually do anything?  If so, should we list what types are supported?

**Comment by davidrapson** [comment]
Good shout re: voiceover. And great catch on the `size` param. It existed in the previous example data, but you are right I don't think it's actually used.

**Comment by davidrapson** [comment]
@davidsauntson Have made both of those updates.

**Review by davidsauntson — APPROVED** [review]
👍

File: testing/features/asset_hyperlink.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@future_release
```
Comment (luke-hill) [inline testing/features/asset_hyperlink.feature:1]: can we pop a `@v2.1.0+` tag on this. So I know when to look at it again.

**Pull request merged**